src/haz3lcore/statics/Info.re

```diff
@@ -128,6 +128,22 @@ type status_variant =
   | Unique
   | Duplicate;
 
+[@deriving (show({with_path: false}), sexp, yojson)]
```

Comment:
was hoping we could have warnings as part of the status type, as another top-level entry alongside InHole and NotInHole. (but not sure if this is right approach), is there a reason this doesn't work out?